Fulfillment/Integration/01_ManageConnections.md

```diff
@@ -0,0 +1,143 @@
+[!!User Interface Connections](../UserInterface/03a_Connections.md)
+
+# Manage the connections
+
+Establish connections via drivers to connect your fulfillers' system with the *Actindo Core1 Platform*. 
+
+In the following, the general settings to create, edit, enable and disable connections are described. For certain connection drivers, you have to setup special configurations. Therefore, the connections below are described separately:
+
+- [Fiege connection](../../FiegeIntegration/Overview/01_General.md)
+
+
+## Create a connection
+
+Create a connection to a certain fulfiller. As soon as a connection has been established, an initial synchronization will be triggered, which will transfer the entire data model of the connected fulfiller mirrored to Actindo. This means that all attributes and attribute sets from the fulfiller are also transferred to Actindo and can be accessed in the *DataHub* module. Depending on the fulfiller to be connected, special configurations may be necessary.
+
+#### Prerequisites
+
+At least one driver has been installed.
+
+#### Procedure
+
+*Fulfillment > Settings > Tab CONNECTIONS*
+
+![Connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/Connections.png "[Connection]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.   
+    The *Create connection* view is displayed.
+
+    ![Create connection](../../Assets/Screenshots/Fulfillment/Settings/Connections/CreateConnection.png "[Create connection]")
+
+2. Enter a name for the connection in the *Name* field.
+
+3. Click the *Driver* drop-down list and select the desired driver. All available drivers are displayed in the list.   
+
+    > [Info] Drivers must be purchased from the main account and then be installed separately in each sandbox or account where you want to use it.   
+
+    The *Credentials* section is displayed below the drop-down list.
+
+    ![Credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/Credentials.png "[Credentials]")
+
+    > [Info] Depending on the selected driver, the fields in the *Credentials* section differ.
+
+
+4. Enter the required data in the fields in the *Credentials* section.
+
+5. Click the [SAVE] button.   
+    The connection will be established. The *Checking credentials...* notice is displayed. After a few seconds, the *Loading data...* notice is displayed.
+
+    ![Checking credentials](../../Assets/Screenshots/Fulfillment/Settings/Connections/CheckingCredentials.png "[Checking credentials]")
+
+    The *Create connection* view is automatically closed when the connection has been established. The *Edit connection* view of the newly created connection is displayed. The *Settings* tab is selected.
```

Comment:
Nur für mein Verständnis:
Gilt das für alle Connections? Oder nur Connections mit Settings? Haben alle Connections in Fulfillment Settings?